### PR TITLE
Fix : Clear filter and add barrel import

### PR DESCRIPTION
**Pull request by anuj-kumary**
In this PR I fixed some issues:-
- Clear filter functionality
- Wishlist Item remove functionality
- Used barrel import

Preview:- https://deploy-preview-10--eaglestores-react.netlify.app/

### CODE REVIEW

**Review by Rohanmond — APPROVED** [review]
looks good to merge

**Review by samyakshah3008 — APPROVED** [review]
Nice refactoring ✅ 
Good for approval

File: eagle-stores/src/App.js [hunk 1]
```diff
@@ -1,13 +1,10 @@
 import './App.css';
 import { Routes, Route, Navigate } from 'react-router-dom';
-import { Home, Login, Product, Cart, WishList } from './pages';
-import { Footer, NavBar } from './components';
+import { Home, Login, Signup, Product, Cart, WishList, Logout } from './pages';
```
Comment (Shah-Arsalan) [inline eagle-stores/src/App.js:3]: Great to see that you have used barrel exports

File: eagle-stores/src/pages/Product/components/ProductCard/components/Product.jsx [hunk 2]
```diff
@@ -39,17 +43,26 @@ export default function Product({ item }) {
...
+        });
       if (response.status === 200 || response.status === 201) {
         dispatch({
           type: ACTION_TYPE.WISHLIST,
           payload: { wishlist: response.data.wishlist },
         });
-        ToastHandler('success', 'Product Added To Wishlist');
       }
+      if (wish)
+        ToastHandler('warn', 'Successfully deleted product from wishlist');
```
Comment (Shah-Arsalan) [inline eagle-stores/src/pages/Product/components/ProductCard/components/Product.jsx:64]: Great to see you have used a toast as well

**Review by Shah-Arsalan — APPROVED** [review]
Code looks neat to me . I would suggest a merge!

**Pull request merged**